tests/ros2/test_msg.py

```diff
@@ -0,0 +1,50 @@
+"""
+license BSD 3-clause
+Copyright (c) 2020, New York University and Max Planck Gesellschaft
```

Comment:
If this file is a copy/paste from something copyrighted NYU / MPI, please provide a link or hint to the initial one: it will be easier to track changes. Otherwise, you're at CNRS now ;)

Reply:
Agreed I need to change that ^^